static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -1,9 +1,4 @@
 $grey: #dbdbdb;
```

Comment:
What about killing this too and replacing it by `$neutral-3` (`neutral-2` is a bit brighter, `neutral-4` is a bit darker).